### ISSUE TITLE
Questions about ML part

### ISSUE THREAD

**Issue by Dantistnfs** [BernsteinLab/aml2019#6]
Hi, I was reading your paper and I have several questions about ML part
1) Inside 1_RFtrain.R I didn't found division on training and validation sets, does it mean you've been checking accuracy on your training set (I understood that you've made CV, but it's nearly not enough to make conclusion that method has 95% accuracy)?
2) I don't see motivation to use two different RF classifiers because of how RF works you're basically passing variable values from outer RF classifier (just in bit altered form) to internal RF classifier.
3) Why don't you limit things like depth and nodesize for RF, because with 1000 trees it will lead to heavy overfitting in relatively a small dataset.
